<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>guiWindow</class>
 <widget class="QMainWindow" name="guiWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Path Controller</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>548</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="Controls_Group">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="title">
       <string>Controls</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="Start_button">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Pause_button">
         <property name="text">
          <string>Pause OFF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Stop_button">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Log_File_Label">
         <property name="text">
          <string>File Prefix</string>
         </property>
         <property name="buddy">
          <cstring>Sensor_File_Save_Box</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Sensor_File_Save_Box">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QVTKWidget" name="qvtkWidget" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
